recipe/meta.yaml

```diff
@@ -0,0 +1,69 @@
+{% set name = "pypdfium2" %}
+{% set version = "4.19.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 1ca3a2ed080c263229af3fbff35ad7f751361861f10893d9908d4d852fe6eb28
+
+build:
+  skip: True  # [linux and (ppc64le or s390x)]
+  entry_points:
+    - pypdfium2 = pypdfium2.__main__:cli_main
+  script_env:
+    - PDFIUM_PLATFORM=none
+  script:
+    - cp bindings/raw.py src/pypdfium2/raw.py
+    - {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - python
+    - setuptools
+    - pip
+    - wheel !=0.38.0,!=0.38.1
+    - ctypesgen-pypdfium2-team 1.1.1+17_g10b2b4
```

Comment:
If using the reference bindings, then you don't need ctypesgen here anymore.
(The wheel code passage is also not used with `PDFIUM_PLATFORM=none`, but it is currently still an import-time requirement. We could change this, though.)